src/main/java/org/mule/service/scheduler/internal/DefaultSchedulerService.java

```diff
@@ -297,6 +304,22 @@ public SchedulerThreadPools load(SchedulerPoolsConfigFactory key) throws Excepti
 
       poolsMaintenanceScheduler = ioScheduler();
       poolsMaintenanceTask = poolsMaintenanceScheduler.scheduleAtFixedRate(() -> poolsByConfig.cleanUp(), 1, 1, MINUTES);
+
+      if (usageTraceIntervalSecs != null) {
+        usageReportingScheduler = cpuLightScheduler();
```

Comment:
Don't think we should be using cpuLight for this, at it will interfere with results.  Either a dedicated thread/scheduler or if not then the `poolsMaintenanceScheduler`.